### PR TITLE
Feature/query update table

### PR DESCRIPTION
**Pull request by PNdunguMaina**
In this project, I did the following ✍️ 

- [ ] Used database transactions
- [ ] Safely modified and deleted records in SQL
- [ ] Prepared complex queries that answered analytical questions
- [ ] Attached are screenshots of the results from the queries
Merged_document

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,3 +7,39 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg >10.5;
...
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+-- day 2
+-- Rename column inside a transaction
+BEGIN TRANSACTION;
+ALTER TABLE animals
+RENAME COLUMN species
+TO unspecified;
+ROLLBACK TRANSACTION;
+
```
Comment (Wes-Isaac) [inline queries.sql]: - [ ] Great job so far after the ROLLBACK check if the change was undone, usually run a select statement.

File: queries.sql [hunk 2]
```diff
@@ -7,3 +7,39 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg >10.5;
...
+-- Write species name with a condition inside a  transaction
+BEGIN WORK;
+UPDATE animals SET species='Digimon' WHERE name Like'%mon%';
+UPDATE animals SET species='Pokemon' WHERE species IS NULL;
+COMMIT WORK;
+
+-- delete data and revert
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (Wes-Isaac) [inline queries.sql]: - [ ] Great job so far after the ROLLBACK check if the change was undone, usually run a select statement.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @PNdunguMaina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great PR description.
- Professional README 👀.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @PNdunguMaina 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**